src/frameworks/vue/template/server/index.js

```diff
@@ -1,9 +1,10 @@
 const express = require('express');
-
-const app = express();
-const http = require('http').Server(app);
 const path = require('path');
 const serveStatic = require('serve-static');
+const http = require('http');
```

Comment:
I'm gonna be nitpicky here. Can these be in alphabetical order?
```suggestion
const http = require('http');
const path = require('path');
const serveStatic = require('serve-static');
```